app/test/fuzz_abci_test.go

```diff
@@ -25,7 +27,12 @@ func TestPrepareProposalConsistency(t *testing.T) {
 		t.Skip("skipping TestPrepareProposalConsistency in short mode.")
 	}
 	encConf := encoding.MakeConfig(app.ModuleEncodingRegisters...)
-	testApp, _ := testutil.SetupTestAppWithGenesisValSet()
+	accounts := make([]string, 10000)
```

Comment:
we might be able to decrease this back to something smaller, which is desired since using this many different accounts makes this test about 10-20 seconds slower locally